plugins/.gitignore

```diff
@@ -5,25 +5,18 @@
 !audit_logs/
```

Comment:
Should I get rid of this file altogether? We no longer copy external plugins into this directory, right?

Reply:
You still need to if you want to run cmake tests for an external plugin.  There may be a way to hack around this limitation, but I haven't explored it yet.